### PR TITLE
New VSS package in autoupdater, updates to other packages

### PR DESCRIPTION
**Pull request by adjackura**
google-compute-engine-auto-updater
+ Add google-compute-engine-vss to autoupdater

google-compute-engine-powershell
+ Rename many functions in to better match PowerShell style, provide aliases for old names
+ Update Write-Log to include a global var of who is doing the logging
+ Delete a few unused functions

### CODE REVIEW

File: sysprep/instance_setup.ps1 [hunk 1]
```diff
@@ -44,9 +44,8 @@ $script:setupcomplete_loc = "$script:setupscripts_dir_loc\SetupComplete.cmd"
 $script:show_msgs = $false
 $script:write_to_serial = $false
 
-# Import Modules
 try {
-  Import-Module $script:gce_base_loc -ErrorAction Stop
+  Import-Module $script:gce_base_loc -ErrorAction Stop 3>$null
```
Comment (illfelder) [inline sysprep/instance_setup.ps1]: Please add a space: `3> $null`.

File: sysprep/sysprep.ps1 [hunk 2]
```diff
@@ -83,7 +84,7 @@ if ($help) {
 
 # Import Modules
 try {
-  Import-Module $PSScriptRoot\gce_base.psm1 -ErrorAction Stop
+  Import-Module $PSScriptRoot\gce_base.psm1 -ErrorAction Stop 3>$null
```
Comment (illfelder) [inline sysprep/sysprep.ps1]: And here.

File: gce_base.psm1 [hunk 3]
```diff
@@ -680,6 +592,11 @@ function Set-LogFile {
 
 
 # Export all modules.
+New-Alias -Name _WriteToSerialPort -Value Write-SerialPort
```
Comment (ning-yang) [inline gce_base.psm1]: why do we still need the old function name?

Comment (adjackura) [inline gce_base.psm1]: There are other tools I've seen that use this library

**Pull request merged**